salt/repo/map.jinja

```diff
@@ -0,0 +1,25 @@
+{%- set defaults = salt['grains.filter_by']({
+  'Linux': {
+    'online_mode': True,
+    'runc': {
+      'version': 'latest'
+    },
+    'containerd': {
+      'version': 'latest'
+    },
+    'kubernetes': {
```

Comment:
I get why this is called `kubernetes`, in a way. However, shouldn't this be called `kubelet`, which is defaulted from some higher-level `kubernetes` version value? The reason is that, when upgrading, the version of `kubernetes` (i.e. contol plane) and `kubelet` on various nodes may not be the same.

Reply:
During an upgrade, you don't upgrade everything all at once, there's a specific order in which to do things.

However, I guess we can figure that out when the case arises, then split version variables or whatever. So not necessarily any changes to be made right now.